docs/Text/README.md

```diff
@@ -30,3 +30,32 @@ BpkText(context).apply {
    weight = BpkText.Weight.EMPHASIZED
 }
 ```
+
+
+## Theme Props
+
+- `fontFamilyBase`
+- `fontFamilyEmphasized`
+- `fontFamilyHeavy`
+
+BpkText accepts the above themeable attributes for the following styles
```

Comment:
I would change this to follow the same convention of other doc files. E.g. `styles can be set globally through...`